src/models/dashboard-items.js

```diff
@@ -468,7 +496,7 @@ const tableConstructor = ({
         dataElementGroupSetDimensions: [],
         attributeDimensions: [],
         translations: [],
-        filterDimensions: ["ou", antigenCategory],
+        filterDimensions: ["ou", antigen ? antigenCategory : null],
```

Comment:
It seems to work with null values in the array, but maybe it's safer to clean it with `_.compact`